lib/views/create-bug.js

```diff
@@ -157,7 +152,7 @@ function submitBug() {
 
   this.app.bugzilla.createBug({
     product: 'Firefox OS',
-    component: value('#component'),
+    component: (process.env.TEST ? 'GAIA' : value('#component')),
```

Comment:
As someone looking at this without knowing that, I would be more likely to understand that this is ```component``` field correlates directly with the Bugzilla component if it was cased the same.